kraken/pvc/pvc_scenario.py

```diff
@@ -46,64 +46,50 @@ def run(scenarios_list, config):
-                volumes_list = runcommand.invoke(command, 60).rstrip()
-                volumes_list_json = json.loads(volumes_list)
-                for entry in volumes_list_json:
-                    if len(entry["persistentVolumeClaim"]["claimName"]) > 0:
-                        volume_name = entry["name"]
-                        pvc_name = entry["persistentVolumeClaim"]["claimName"]
+                pod = kubecli.get_pod_info(name=pod_name, namespace=namespace)
+                
+                if pod is None:
+                    sys.exit(1)
+                
+                for volume in pod.volumes:
+                    if volume.pvcName is not None:
+                        volume_name = volume.name
+                        pvc_name = volume.pvcName
+                        pvc = kubecli.get_pvc_info(pvc_name, namespace)
                         break
+                if 'pvc' not in locals():
+                    logging.error(
+                        "Pod '%s' in namespace '%s' does not use a pvc" % (str(pod_name), str(namespace))
+                    )
+                    sys.exit(1)
                 logging.info("Volume name: %s" % volume_name)
                 logging.info("PVC name: %s" % pvc_name)
 
                 # Get container name and mount path
-                command = 'kubectl get pods %s -n %s -o json | jq -r ".spec.containers"' % (
-                    str(pod_name),
-                    str(namespace),
-                )
-                logging.debug("Get mount path command:\n %s" % command)
-                volume_mounts_list = runcommand.invoke(command, 60).rstrip().replace("\n]\n[\n", ",\n")
-                volume_mounts_list_json = json.loads(volume_mounts_list)
-                for entry in volume_mounts_list_json:
-                    for vol in entry["volumeMounts"]:
-                        if vol["name"] == volume_name:
-                            mount_path = vol["mountPath"]
-                            container_name = entry["name"]
+                for container in  pod.containers:
+                    for vol in container.volumeMounts:
+                        if vol.name == volume_name:
+                            mount_path = vol.mountPath
+                            container_name = container.name
                             break
                 logging.info("Container path: %s" % container_name)
                 logging.info("Mount path: %s" % mount_path)
 
                 # Get PVC capacity
-                command = "kubectl describe pvc %s -n %s | grep \"Capacity:\" | grep -Eo '[^: ]*$'" % (
-                    str(pvc_name),
-                    str(namespace),
-                )
-                pvc_capacity = runcommand.invoke(
-                    command,
-                    60,
-                ).rstrip()
-                logging.debug("Get PVC capacity command:\n %s" % command)
+                pvc_capacity = pvc.capacity
```

Comment:
This capacity is a little off from the actual capacity in the PVC 

Can do something similar to what we do to find the Used bytes on the system 
`command = "df %s -B 1024 | sed 1d | awk -F' ' '{print $4}'" % (str(mount_path))`


Output I saw: 
`2022-08-24 21:28:47,466 [INFO] Pod name: kraken-test-pod
2022-08-24 21:28:50,026 [INFO] Volume name: kraken-test-pv
2022-08-24 21:28:50,026 [INFO] PVC name: kraken-test-pvc
2022-08-24 21:28:50,027 [INFO] Container path: kraken-test-container
2022-08-24 21:28:50,027 [INFO] Mount path: /home/krake-dir/
2022-08-24 21:28:50,027 [INFO] PVC capacity: 2097152 KB
2022-08-24 21:28:50,251 [INFO] PVC used: 10307228 KB
2022-08-24 21:28:50,251 [ERROR] 
                        Target fill percentage (50.00%) is lower than current fill percentage (491.49%)
                         or higher than 99%

% oc exec kraken-test-pod -- df /home/krake-dir/ 
Filesystem     1K-blocks     Used Available Use% Mounted on
/dev/nvme0n1p4 523752428 10307356 513445072   2% /home/krake-dir
`

Reply:
Modified this in the latest commit